### PR TITLE
Upgrade to 5.0.441

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

**Comment by Cogitri** [comment]
Seems like 5.0.438 has been released already

**Comment by nedrichards** [comment]
bot, build

File: org.geogebra.GeoGebra.json [hunk 1]
```diff
@@ -40,38 +40,38 @@
...
                 {
                     "type": "extra-data",
-                    "filename": "GeoGebra-Linux-Portable-5-0-432-0.tar.bz2",
+                    "filename": "GeoGebra-Linux-Portable-5-0-438-0.tar.bz2",
                     "only-arches": ["x86_64", "i386"],
-                    "url": "https://download.geogebra.org/installers/5.0/GeoGebra-Linux-Portable-5-0-432-0.tar.bz2",
-                    "sha256": "4055d3175eba7fe386ed802858bb9353b774ca828816e043e87cf21ec24fdf85",
+                    "url": "https://download.geogebra.org/installers/5.0/GeoGebra-Linux-Portable-5-0-438-0.tar.bz2",
+                    "sha256": "32026254db67889b17b30f70151d7de9a7b86ba7c3e15f0150ba5b60eb0e36b9",
                     "size": 136346301
```
Comment (nedrichards) [inline org.geogebra.GeoGebra.json]: you haven't updated the size here. this needs to be accurate as well for extra-data.

run `ls -l` in the directory you downloaded this in - in the same way you ran `sha256sum` to get the sha.

**Comment by Cogitri** [comment]
I feel like it'd be a good idea to squash those commits into one

**Comment by Cogitri** [comment]
5.0.441 has been released

**Comment by nedrichards** [comment]
@Cogitri if it's possible to rebuild the whole thing with the same functionality then absolutely we should do that on Flathub. Very happy to review a PR - we have build resources to get that done.

**Comment by Cogitri** [comment]
Alright, I'll look into it and submit something soon-ish :)

**Pull request merged**